apps/web/src/app/domain/pages/featured/articles/articles.page.html

```
@@ -0,0 +1,90 @@
+            rel="noopener noreferrer"
+            class="group focus-visible:ring-ring focus-visible:ring-offset-background flex flex-col gap-3 rounded-lg outline-hidden focus-visible:ring-2 focus-visible:ring-offset-2"
+          >
+            <div class="bg-muted relative aspect-video overflow-hidden rounded-lg border">
+              @if (article.coverFailed) {
+                <div class="flex h-full w-full flex-col items-center justify-center gap-3 px-6">
+                  <img src="/images/zard.svg" alt="" aria-hidden="true" class="h-10 w-auto opacity-40 dark:invert" />
+                  <p class="text-muted-foreground line-clamp-2 text-center text-sm font-medium">{{ article.title }}</p>
+                </div>
+              } @else {
+                <img
+                  [src]="article.cover"
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)